_locales/_untranslated_en.json

```diff
@@ -23,5 +23,41 @@
   },
   "pref_current_account": {
     "message": "Current Account"
+  },
+  "onboarding_create_instant_account": {
+    "message": "Let's get started!"
+  },
+  "onboarding_alternative_logins": {
+    "message": "I already have a login"
+  },
+  "manual_account_setup_option": {
+    "message": "Manual login"
+  },
+  "instant_onboarding_title": {
+    "message": "Your profile"
+  },
+  "instant_onboarding_create": {
+    "message": "Create profile"
+  },
+  "instant_onboarding_default_info": {
+    "message": "Your profile will be created on the default instance."
```

Comment:
we also thought about using the other wording "Your profile will be created on the following instance: nine.testrun.org" - would have been even easier, as nothing special is done.

however, this will raise questions: what is an instance? what is nine.testrun.org? can i trust that? etc. therefore, unless it seems _really_ needed for ppl "just want to chat with this app" - without knowing or caring instances, chatmail, let alone "nine", it seems advisable to leave that out. the less questions popping up, the better.

and: there is a help button just at the side: we aim to explain things in detail there, for ppl that are interested.